### PR TITLE
[src/components] removing directive PGSQL_VERSION > 94

### LINKED ISSUE #1948: PostgreSQL 9.5 is no longer supported components
Refer to #1938 for instructions and details.

### PR DESCRIPTION
**Pull request by vicilanaya**
Fixes #1948

### CODE REVIEW

**Comment by robe2** [comment]
@cvvergara  Can you check on this one. I think all the lines should be:

funcctx->max_calls = result_count;

But the original code wasn't right (as both the if and endif were the same)

File: src/components/articulationPoints.c [hunk 1]
```diff
@@ -114,11 +114,8 @@ PGDLLEXPORT Datum _pgr_articulationpoints(PG_FUNCTION_ARGS) {
                 &result_count);
 
 
-#if PGSQL_VERSION > 94
         funcctx->max_calls = (uint32_t)result_count;
```
Comment (cvvergara) [inline src/components/articulationPoints.c:117]: Should be:
```
funcctx->max_calls = result_count;
```

File: src/components/biconnectedComponents.c [hunk 2]
```diff
@@ -115,11 +115,8 @@ PGDLLEXPORT Datum _pgr_biconnectedcomponents(PG_FUNCTION_ARGS) {
                 &result_count);
 
 
-#if PGSQL_VERSION > 94
         funcctx->max_calls = (uint32_t)result_count;
```
Comment (cvvergara) [inline src/components/biconnectedComponents.c:118]: Should be:
```
funcctx->max_calls = result_count;
```

File: src/components/bridges.c [hunk 3]
```diff
@@ -113,11 +113,8 @@ PGDLLEXPORT Datum _pgr_bridges(PG_FUNCTION_ARGS) {
                 &result_count);
 
 
-#if PGSQL_VERSION > 94
         funcctx->max_calls = (uint32_t)result_count;
```
Comment (cvvergara) [inline src/components/bridges.c:116]: Should be:
```
funcctx->max_calls = result_count;
```

File: src/components/connectedComponents.c [hunk 4]
```diff
@@ -115,11 +115,8 @@ PGDLLEXPORT Datum _pgr_connectedcomponents(PG_FUNCTION_ARGS) {
                 &result_count);
 
 
-#if PGSQL_VERSION > 94
         funcctx->max_calls = (uint32_t)result_count;
```
Comment (cvvergara) [inline src/components/connectedComponents.c:118]: Should be:
```
funcctx->max_calls = result_count;
```

File: src/components/strongComponents.c [hunk 5]
```diff
@@ -115,11 +115,8 @@ PGDLLEXPORT Datum _pgr_strongcomponents(PG_FUNCTION_ARGS) {
                 &result_count);
 
 
-#if PGSQL_VERSION > 94
         funcctx->max_calls = (uint32_t)result_count;
```
Comment (cvvergara) [inline src/components/strongComponents.c:118]: Should be:
```
funcctx->max_calls = result_count;
```

**Review by cvvergara — CHANGES_REQUESTED** [review]

**Comment by cvvergara** [comment]
Closing in favor of #2006

**Pull request closed without merging**